### ISSUE TITLE
Publish to Hex.pm

### ISSUE THREAD

**Issue by waisbrot** [project-fifo/rebar3_lint#1]
Could you publish this to Hex? I was searching for "elvis" there to see if someone had already made such a plugin.

**Comment by Licenser** [comment]
Absolutely it's work in progress. This is the blocker https://github.com/inaka/elvis_core/issues/7 since hex packages require all dependencies to be hex packages themselves.

**Comment by waisbrot** [comment]
Inaka's removed the issues link from elvis_core...

Anyway, all the dependencies of elvis_core are now in Hex. I tried building it, though, and wasn't able to fetch katana because of some mysterious problem that I can't figure out (hexpm/rebar3_hex#29). If katana fetched correctly, though, then elvis_core could get into Hex.

**Comment by tsloughter** [comment]
The issue remains. Katana is only published and buildable for `erlang.mk`.

**Comment by tsloughter** [comment]
@waisbrot also it isn't exactly that `erlang.mk` deps can't be built by rebar3 but because in many cases that is true (and katana's package happens to be one of those cases) we do filter them out, we do the same for `mix` packages.

**Comment by Licenser** [comment]
Why the heck would they remove the issue -.-

**Comment by waisbrot** [comment]
They removed the whole "issues" link from the repository. I don't _think_ it was directed at you @Licenser :-)

**Comment by Licenser** [comment]
oh silly, well hard to keep track of it -.- but it seems the package while there partially broken

**Comment by elbrujohalcon** [comment]
`elvis_core` issues are tracked in http://github.com/inaka/elvis/issues, sorry about the inconveniences.

**Comment by waisbrot** [comment]
Like I said in #2, I'm able to build this project using only Hex dependencies now and it appears to work. What problems are you seein @Licenser ?

**Comment by Licenser** [comment]
this fails for me with the following error:
```
Schroedinger:rebar3_lint heinz [master] $ rebar3 compile
===> Fetching rebar3_hex ({pkg,<<"rebar3_hex">>,<<"1.19.0">>})
===> Downloaded package, caching at /Users/heinz/.cache/rebar3/hex/default/packages/rebar3_hex-1.19.0.tar
===> Compiling rebar3_hex
===> Verifying dependencies...
===> Package inaka_mixer-0.1.4 not found. Fetching registry updates and trying again...
===> Updating package registry...
===> Writing registry to /Users/heinz/.cache/rebar3/hex/default/registry
===> Generating package index...
===> [cloudi_service_oauth1:1.5.1] Only existing version of cloudi_service_db_riak is 1.3.3 which does not match constraint ~> 1.5.1. Using anyway, but it is not guaranteed to work.
===> Writing index to /Users/heinz/.cache/rebar3/hex/default/packages.idx
===> Package not found in registry: inaka_mixer-0.1.4.
```

**Comment by IgorKarymov** [comment]
@Licenser try to update rebar3 to the latest version (I had a similar issue in another my project).

**Comment by Licenser** [comment]
No it's not a issue with rebar it's a issue with the package, it claims to be version X in the app.src while announcing itself as Y for hex/rebar that is what freaks it out.

**Comment by Licenser** [comment]
Hi the upstream packages are now all wokring, I published 0.1.7 to hex, enjoy!
